src/frontend/src/pages/FinancePage/FinancePage.tsx

```diff
@@ -41,6 +42,10 @@ const FinancePage = () => {
 
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
 
+  const [AccountCreditModalShow, setAccountCreditModalShow] = useState<boolean>(false);
+  const handleAccountCreditClose = () => setAccountCreditModalShow(false);
+  const handleAccountCreditOpen = () => setAccountCreditModalShow(true);
```

Comment:
you can just pass these in to onClick on handleClose directly without making them helper method